packages/gatsby/README.md

```diff
@@ -22,6 +22,13 @@ module.exports = {
       options: {
         // public API Key
         publicAPIKey: 'MY_PUBLIC_API_KEY',
+        mapEntryToGatsbyConfig: (entry) => {
```

Comment:
Thanks @apaniel90vp for the pr, looks good, what do you think of naming this `mapEntryToContext`  or any similar wording that indicates that this adds to the maps fields from entry to the page context.

Reply:
Totally, makes more sense, done @teleaziz